src/main/java/de/lh/tool/config/security/JwtAuthenticationFilter.java

```diff
@@ -44,7 +48,7 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 				SecurityContextHolder.getContext().setAuthentication(authentication);
 			}
 		} catch (Exception ex) {
-			logger.error("Could not set user authentication in security context", ex);
+			log.error("Could not set user authentication in security context", ex);
 		}
 
 		filterChain.doFilter(request, response);
```

Comment:
In getJwtFromRequest():
Länge von "Bearer " vllt nicht hardkodieren.
Eine Alternative à la mglw.:
```
private String getJwtFromRequest(HttpServletRequest request) {
		final String BEARER_PREFIX = "Bearer ";
		String bearerToken = request.getHeader("Authorization");
		if (StringUtils.hasText(bearerToken) && bearerToken.startsWith(BEARER_PREFIX)) {
			return bearerToken.substring(BEARER_PREFIX.length(), bearerToken.length());
		}
		return null;
	}
```